packages/application-generic/src/usecases/add-job/add-job.usecase.ts

```diff
@@ -102,12 +114,7 @@ export class AddJob {
       })
     );
 
-    if (delay === null) {
-      Logger.warn(
-        'Variable delay is null which is not apart of the definition',
```

Comment:
this should not be a possible step because we return on each amount === undefined.